content/02.results.md

```diff
@@ -58,12 +44,40 @@ A review process identified samples that could not be explained parsimoniously,
 
 A further 82 samples were excluded as potential sample mix ups.
 
-Following all sample QC steps, 3,483 samples (74.2%) were retained from the original cohort for analysis.
+Following all sample QC steps, 3,483 wild-caught samples (74.2%) were retained from the original cohort for analysis.
+
+This represents an additional 1,823 mosquitoes relative to the phase 2 release.
+
+Due to a change in assessment of sample quality where technical replicates are available, the preferred replicate was changed for 172 mosquitoes between phase 2 and phase 3.
+
+9 samples included in phase 2 are not present in this release (sup XXX).
+
+The AG1000G-X submission set, made up of laboratory experimental crosses, was subject to a slightly different QC process.
+
+Firstly an analysis based on rates of Mendelian error identified true fathers of crosses (where multiple males were introduced to cages), and validated provided pedigrees.
+
+Of the 7XX samples provided we were able to validate 15 crosses to a high level of confidence, comprising 299 samples.
+
+4 of these crosses are novel relative to phase 2.
+
+These samples went through a modified sequence quality assurance process, where 1 sample was removed for insufficient coverage (methods).
```

Comment:
Ah, maybe this explains the number of samples passing QC. There are 2 samples that fail the contamination filter, one of which is a parent. IIRC, the conclusion of previous discussion was to let the parent PASS (ME shows it is fine), but the offspring would stay FAIL.